packages/@dcl/sdk/src/index.ts

```diff
@@ -18,6 +19,12 @@ export async function onUpdate(deltaTime: number) {
  * Function that is called before the first update and after the evaluation of the code.
  */
 export async function onStart() {
+  const compositeProvider = await createContentFetchCompositeProvider()
+  const mainComposite = compositeProvider.getCompositeOrNull('main')
+  if (mainComposite) {
+    instanceComposite(engine, mainComposite, () => engine.addEntity(), compositeProvider)
+  }
+
   await engine.seal()
 
   const response = await crdtGetState({ data: new Uint8Array() })
```

Comment:
The composite instantiation should only happen if the renderer does not have any "scene owned entity" to report.

https://github.com/decentraland/protocol/blob/d0a21d73e4d50e6d94f161269c021688ef719083/proto/decentraland/kernel/apis/engine_api.proto#L187

```suggestion
  const compositeProvider = await createContentFetchCompositeProvider()

  await engine.seal()

  const response = await crdtGetState({ data: new Uint8Array() })
  
  if (!response.hasEntities) {
    const mainComposite = compositeProvider.getCompositeOrNull('main')
    if (mainComposite) {
      instanceComposite(engine, mainComposite, () => engine.addEntity(), compositeProvider)
    }
  }
```

Reply:
I'm temporarily not adding this because first, we need to discriminate the previously stashed data from the renderer.
Now, `hasEntities` is always true because of the initialization of `PlayerEntity` and `CameraEntity`.
Also, the renderer is not storing the non-core components (custom components)
cc @pbosio